examples/kinematic/potential_energy_from_mass_and_height.py

```diff
@@ -0,0 +1,27 @@
+from symplyphysics import (
+    units, convert_to,solve, SI
+)
+from symplyphysics.laws.kinematic import potential_energy_from_mass_and_height as potential_energy
+
+#A bullet weighing 9 grams is fired from the gun vertically upwards and reaches
+#an altitude of 1,500 meters. Under the action of gravity it falls freely
+#to the ground and hits a shard of glass lying on the ground
+#Can a bullet break a piece of glass on the ground if it takes 100 joules of work ?
```

Comment:
Breaking glass does not depend on work. It depends of pressure, I guess.
I'll think how to improve this example.

Reply:
For creating pressure need some work, really?

Reply:
Yes, but I guess it depends on force. Work is more about dissipating heat or moving things. Breaking is another process.